src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -14,11 +14,13 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
     "select u from SessionTemplate u " +
       "where u.prison.code = :prisonCode " +
       "and (u.validToDate is null or u.validToDate >= :firstBookableDay) " +
-      "and (u.validFromDate <= :lastBookableDay)"
+      "and (u.validFromDate <= :lastBookableDay) " +
+      "and (:enhancedPrivilege = true or u.enhanced = false)"
```

Comment:
Sorry didn't understand this one. Should it be u.enhanced = : enhancedPrivilege?

Reply:
You had a similar question on similar code a while back,... basically brings back all session templates if in enhance templates are requested otherwise just session templates that are not enhance

Reply:
will call this inclEnhancedPrivilegeTemplates